### PR TITLE
Enable configuring custom healthcheck url.

### PR DESCRIPTION
**Pull request by maciejwalkowiak**

### CODE REVIEW

**Comment by yana-aus-korea** [comment]
Build triggered. sha1 is merged.

**Comment by yana-aus-korea** [comment]
Build started sha1 is merged.

**Comment by yana-aus-korea** [comment]
Build finished.

File: lib/baustelle/config/application.rb [hunk 1]
```diff
@@ -36,6 +36,10 @@ def elb_visibility
         elb.fetch('visibility', 'public')
       end
 
+      def healthcheck_url
```
Comment (suxor42) [inline lib/baustelle/config/application.rb]: In this case the name should be healthcheck path because the value is not an URL.

**Pull request merged**